corehq/apps/locations/bulk_management.py

```diff
@@ -902,7 +906,11 @@ def handle_delete(loc):
         with transaction.atomic():
             for loc in stubs:
                 if loc.do_delete:
-                    handle_delete(loc)
+                    if loc.is_new:
+                        if excel_importer:
+                            excel_importer.add_progress()
```

Comment:
What's going on here?  It adds progress an extra time if a location is both new and flagged "delete"?

Reply:
This is not new, I've just changed the place where it's happening. It used to happen in the loop where locations were being deleted:

https://github.com/dimagi/commcare-hq/blob/3f0a59bf228d14370409365f702f80772a158f49/corehq/apps/locations/bulk_management.py#L923-L926

A new location that's also being deleted is effectively a no-op, but it's still counted on the progress bar.